translations/en.json

```diff
@@ -301,7 +301,9 @@
     "verificationResult": "Verification result",
     "verified": {
       "title": "Parking cards added",
-      "text": "Your parking cards email **{{email}}** has been added. You can see your cards in the Parking cards section.",
+      "text_empty": "Your parking cards email **{{email}}** has been added. You can see your cards in the Parking cards section.",
```

Comment:
I think it's very wrong to use the same syntax for custom keys and for build in behaviour (`_empty` signifies that it's a built-in syntax handling plurals...)